.gitignore

```diff
@@ -39,3 +39,6 @@ tests/s3/test_s3
 tests/*/.depend
 tests/*/*.debug
 tests/*/*.full
+# python tests
+tests/python/__pycache__
+tests/python/kivaloo/__pycache
```

Comment:
Is it intentional that one of these is `__pycache__` and the other is `__pycache` ?

Reply:
oops